scripts/zkai_d128_proof_artifact_backend_spike_gate.py

```
@@ -0,0 +1,882 @@
+            error = str(err) or f"{err.__class__.__name__} with empty message"
+            results.append(
+                {
+                    "index": index,
+                    "mutation": mutation,
+                    "surface": surface,
+                    "mutated_accepted": False,
+                    "rejected": True,
+                    "rejection_layer": surface,
+                    "error": error,
+                }
+            )
```

Ruff (0.15.12)

[warning] `try`-`except` within a loop incurs performance overhead

(PERF203)